packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -373,18 +345,10 @@ class FlameChartItem {
     _labelWrapper.append(_label);
     e.append(_labelWrapper);
 
-    final style = e.style;
-    style
+    e.style
       ..background = colorToCss(_backgroundColor)
-      ..left = '${flameChartInset + _startingLeft}px';
-    if (_startingWidth != null) {
-      style.width = '${_startingWidth}px';
-      // This is critical to avoid having labels overflow the items boundaries.
-      // For some reason, overflow:hidden does not play well with
-      // position: sticky; so we have to implement this way.
-      _labelWrapper.style.maxWidth = '${_startingWidth}px';
-    }
-    style.top = '${_top}px';
+      ..top = '${_top}px';
+    updateHorizontalPosition(zoom: 1);
```

Comment:
the horizontal position logic was duplicated in a couple places so I moved it to a helper as I needed to update it to tweak the maxWidth to avoid slight text overflow.